frontend/src/lib/i18n/en.json

```diff
@@ -111,12 +111,39 @@
       "electionManifesto": "Election Manifesto",
       "tapToAddPhoto": "Tap to add photo"
     },
+    "setPassword": {
+      "greeting": "Hello, {name}!",
+      "description": "Set a secure password according to the requirements below:",
+      "password": "Password",
+      "confirmPassword": "Confirm password",
+      "setPassword": "Set password",
+      "passwordsDontMatch": "Passwords don't match",
+      "passwordNotValid": "Password not valid",
+      "registrationError": "Error registering password"
+    },
+    "passwordValidation": {
+      "length": "At least {minPasswordLength} characters",
+      "uppercase": "Uppercase letter",
+      "lowercase": "Lowercase letter",
+      "number": "Number",
+      "symbol": "Symbol",
+      "username": "Avoid using your name in the password",
+      "repetition": "Contains character repetition",
+      "commonPatterns": "Contains common sequences or patterns"
+    },
     "genders": {
       "male": "Male",
       "female": "Female",
       "nonBinary": "Non-binary",
       "other": "Other",
       "preferNotToSay": "Prefer not to say"
+    },
+    "registration": {
+      "enterCode": "Enter the passkey you’ve received from your party to edit your data and opinions.",
```

Comment:
passkey => registration code, I'd also change the ending to be more related to registering yourself to emphasize it's not one-time